loculus_values/values.yaml

```diff
@@ -435,6 +435,10 @@ defaultOrganismConfig: &defaultOrganismConfig
         header: Authors
         truncateColumnDisplayTo: 15
         ingest: ncbiSubmitterNames
+        preprocessing:
```

Comment:
sadly this means that while we are waiting for the sequences to be reprocessed the old pipeline will not be usable (throw an error due to unknown function) unless we temporarily add this to the preprocessing pipeline v1 as a no-op.

Reply:
aah, good catch! dying think it's a problem since the probability of submission is low no?

Reply:
yeah probably I created this just now as a workaround: https://github.com/loculus-project/loculus/pull/3065 but potentially we actually do want submissions to not work in the off-chance someone submits with incorrect authors format while we are doing this